sass/elements/_schedule.scss

```diff
@@ -0,0 +1,90 @@
+.schedule-blocks-regular {
+    background-color: $primary-color;
```

Comment:
I believe we are using 2 spaces for scss, however I am open to changing this to 4 if thats preferred. The .editorconfig should help with this (see the readme)